hackertracker/HTUpdatesViewController.swift

```diff
@@ -125,6 +131,20 @@ extension HTUpdatesViewController : UITableViewDataSource, UITableViewDelegate
         //Only make the easter egg visible on top overscroll
         skullBackground.isHidden = scrollView.contentOffset.y > 0
         logoCenterToTopMargin.constant =  ((updatesTableView.contentInset.top + 64) / 2.0) - ((((updatesTableView.contentInset.top + 64) / 2.0) - 37) * percentage)
+
+        if scrollView.contentOffset.y < -500 {
```

Comment:
You want to do this based on the contentInset minus a set value. The iPad could already have a content offset of -500